### PR TITLE
237 hide toolbar button captions

### PR DESCRIPTION
**Pull request by rexius41**
## Checklist

_Confirm you have completed the following actions prior to submitting this PR._

 - [x] There is an existing issue report for this PR.
 - [x] I have forked this project.
 - [x] I have created a feature branch.
 - [x] My changes have been committed.
 - [x] I have pushed my changes to the branch.

## Title

Adding option to hide text in buttons in toolbar

## Description

Hiding the text in toolbar is made by collapsing textblock, and showing tooltip on the buttons. Setting option is in settings/advanced/crawler.

## Issue Resolution

Option to hide toolbar button captions #237

This Pull Request Fixes #

## New or Changed Features

Option to hide toolbar buttons text and while hidden all buttons have the same width

### CODE REVIEW

File: src/TumblThree/TumblThree.Presentation/Properties/Resources.resx [hunk 1]
```diff
@@ -1228,4 +1228,13 @@ The cookies for the privacy consent are saved and the process needs to be done o
...
     <value>When an authorization error occurs, the app tries a new way of handling it in order to continue.</value>
   </data>
+  <data name="HideToolBarButtonsText" xml:space="preserve">
+    <value>Hide toolbar button's text  *app restart required</value>
+  </data>
+  <data name="ToolTipHideToolBarButtonsText" xml:space="preserve">
+    <value>Hide toolbar button's text from main application window</value>
+  </data>
+  <data name="ToolTipHideToolBarButtonsTextDescription" xml:space="preserve">
+    <value>Hides(colapses) the text on buttons from main application windows and leaves tooltips on so the toolbars buttons have all the same footprint</value>
```
Comment (thomas694) [inline src/TumblThree/TumblThree.Presentation/Properties/Resources.resx:1238]: Hides the text on buttons from main application window and leaves tooltips on so all the toolbar buttons have the same width
```suggestion
    <value>Hides the text on buttons from main application window and leaves tooltips on, so all the toolbar buttons have the same width</value>
```

File: src/TumblThree/TumblThree.Presentation/Properties/Resources.de.resx [hunk 2]
```diff
@@ -979,4 +979,13 @@ Die zur Authentifizierung verwendeten Cookies werden gespeichert und der Authent
...
     <value>Wenn ein Autorisierungsfehler auftritt, versucht die App eine neue Methode zur Behebung, um fortzufahren.</value>
   </data>
+  <data name="HideToolBarButtonsText" xml:space="preserve">
+    <value>Text der Symbolleiste ausblenden *Neustart der Anwendung erforderlich</value>
+  </data>
+  <data name="ToolTipHideToolBarButtonsText" xml:space="preserve">
+    <value>Text der Symbolleiste aus dem Hauptanwendungsfenster ausblenden.</value>
+  </data>
+  <data name="ToolTipHideToolBarButtonsTextDescription" xml:space="preserve">
+    <value>Verbirgt (kollabiert) den Text auf den Schaltflächen im Hauptanwendungsfenster und lässt Tooltips aktiviert, damit die Schaltflächen der Symbolleiste alle denselben Fußabdruck haben.</value>
```
Comment (thomas694) [inline src/TumblThree/TumblThree.Presentation/Properties/Resources.de.resx:989]:
```suggestion
    <value>Blendet den Text auf den Schaltflächen im Hauptfenster der Anwendung aus und lässt die QuickInfos eingeschaltet, so dass alle Schaltflächen in der Symbolleiste die gleiche Breite haben.</value>
```

**Review by thomas694 — CHANGES_REQUESTED** [review]
Thanks, it looks good.
I made a few suggestions, for convenience, you can mostly click "Add suggestion".

File: src/TumblThree/TumblThree.Presentation/Views/SettingsView.xaml
[the remark this answers is not in the document]
Comment (rexius41) [reply src/TumblThree/TumblThree.Presentation/Views/SettingsView.xaml:1]: will move it there, accidently clicked the review button before making this change

[the remark this answers is not in the document]
Comment (rexius41) [reply src/TumblThree/TumblThree.Presentation/Views/SettingsView.xaml:1]: Added it to the General tab as the last checkbox

File: src/TumblThree/TumblThree.Presentation/Views/CrawlerView.xaml
[the remark this answers is not in the document]
Comment (rexius41) [reply src/TumblThree/TumblThree.Presentation/Views/CrawlerView.xaml:1]: Done it plus changed the tooltip that is shown only when the text is hiden so it does not obstruct user when he has text in the button

File: src/TumblThree/TumblThree.Applications/ViewModels/SettingsViewModel.cs [hunk 3]
```diff
@@ -8,6 +8,7 @@
 using System.Threading.Tasks;
 using System.Waf.Applications;
 using System.Waf.Applications.Services;
+using System.Waf.Foundation;
```
[the remark this answers is not in the document]
Comment (rexius41) [reply src/TumblThree/TumblThree.Applications/ViewModels/SettingsViewModel.cs]: removed not necessery/autofill

**Review by thomas694 — APPROVED** [review]
Thanks for your PR.

**Pull request merged**